ecl/hqlcpp/hqlcpp.cpp

```diff
@@ -12039,7 +12039,7 @@ void HqlCppTranslator::buildScriptFunctionDefinition(BuildCtx &ctx, IHqlExpressi
                 args.append(*createActualFromFormal(queryParam));
             else
                 args.append(*LINK(bodyCode->queryChild(0)));
-            args.append(*createConstant(createUtf8Value(fieldlist.length()-1, fieldlist+1, makeUtf8Type(UNKNOWN_LENGTH, NULL))));
+            args.append(*createConstant(createUtf8Value(fieldlist.length()-1, fieldlist.str()+1, makeUtf8Type(UNKNOWN_LENGTH, NULL))));
```

Comment:
This is the line which caused the build to fail.  I think windows is incorrectly using an explicit boolean cast operator.